util/src/main/i18n/templates/misc.properties

```diff
@@ -120,6 +120,12 @@ misc.or = {0} or {1}
 # do not translate, keep as-is
 misc.snowman = ☃
 
+# do not translate, keep as-is
+misc.checkmark = ✔
+
+# do not translate, keep as-is
+misc.crossmark = ✘
```

Comment:
We don't need to translate these, let's put them as static `TextComponent` constants.

Reply:
Where would these go?